### ISSUE TITLE
✨ Feature Suggestion | In-site issue tracking, FAQ and transparency

### ISSUE THREAD

**Issue by chloelovesdev** [privacytools/privacytools.io#2104]
## Description

I believe the site should implement some sort of display of the issues the site has had, and has, for transparency and therefore further inspection.
Some sort of link or iframe with the GitHub issue tracker would suffice, but a designed integration would be better. Also, there is no direct link to the issues with the site on it.

## Why is this a problem

If there is a problem, or has been a problem with the site, it should be on there. This helps users understand what is wrong or could be wrong with what we are saying. It also helps further the thought processes behind the selection of our services, and encourages further contribution (they now know why we have selected these for privacy instead of trusting them blindly). Both help establish trust.

Each service implemented should include a link to it's issue on the GitHub tracker, or an explanation of why it was selected. Source code analysis (if required) and similar auditing techniques could help boost trust in the privacy tools selected, but this should be a seperate issue.

## FAQ

A FAQ would help users see existing solutions to problems that have come up with the site. For example, "Why hasn't PrivacyTools made it's own privacy-focused distribution?"
This could be done already with the issue tracker, however, there are certain questions that wouldn't suffice in the tracker or may be hidden away under another "closed" section. Not every question someone has will be on the tracker, and not every [simple] question may be easily found, unless you go through all 100 pages.

## Conclusion

More explanations and proofs of why they are selected for *privacy* need to be found on the website. More transparency needs to be integrated into the actual site, instead of digging through a list of "issues".

**Comment by wylel** [comment]
I agree. I know when I first joined I saw sections being removed, and wasn't fully sure where to find information on why that happened. Maybe a "recently removed" section on the site showing X amount of previously removed recommendations with a brief summary as to why would be good.

Open for discussion, and maybe even putting in some PRs frequently for that.

**Comment by lynn-stephenson** [comment]
We are going to adopt [semantic versioning](https://semver.org/), a slightly different software development model, and use GitHub's releases feature. You can be notified of releases with, or without a GitHub account. Such as using a feeder application. Each release will indicate any changes to the project, with a short description and links to related issues.

GitHub has Atom feeds for releases: https://github.com/privacytools/privacytools.io/releases.atom
